1.raspberry-pi-stuff/octoberry/octoberry/__main__.py

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env python3
 import sys
-import re
-from pygame import mixer
+import subprocess
 from urllib.parse import urlparse
 import os
 import requests
+from typing import Union, Tuple
+import logging
+import keyboard
 
-BACKEND_URL = "http://localhost:5000" # TODO: change this to the backend url
+BACKEND_URL = "https://8bd6-2602-fb65-0-100-bd72-510e-3958-389.ngrok-free.app" # TODO: change this to the backend url
 
-def play_sound(sound_url, cached_sound_path: None | str=None) -> None:
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def _play_sound(sound_url, cached_sound_path: Union[None, str]=None) -> None:
     '''
     Play sound from url, if cached_sound_path is provided it will be used to cache the sound
```

Comment:
the doc isn't correct - 
we will always try to find a cached mp3. this parameter only allows us to have a different than default value for it